static/js/components/GcnEventPage.jsx

```diff
@@ -39,6 +39,8 @@ import GcnSelectionForm from "./GcnSelectionForm";
 import ObservationPlanRequestForm from "./ObservationPlanRequestForm";
 import ObservationPlanRequestLists from "./ObservationPlanRequestLists";
 
+import RenderTags from "./RenderTags";
```

Comment:
Since this is specific to Gcn, consider including Gcn in the component name e.g. `GcnTags`.

Reply:
Good call! Fixed!